src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-5121141.mdx

```diff
@@ -3,6 +3,9 @@ subject: Python agent
 releaseDate: '2020-05-04'
 version: 5.12.1.141
 downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: []
+bugs: ['AIOHTTP http client urls displayed as unknown.url when using YARL URLs', 'Data may not be recorded when tornado's RequestHandler.finish', 'asyncio.ensure_future / asyncio.create_task is unable to spawn new transactions', 'py3: A runtime instrumentation error may have occurred when transactions were finalized', 'Application crash when starting a transaction when one is already active', 'Creating a message transaction with distributed tracing enabled may have resulted in a crash']
```

Comment:
i think we might need to get rid of or escape the `:` here, the frontmatter gets parsed as `yaml`